__tests__/e2e/countries.spec.ts

```diff
@@ -46,9 +46,11 @@ describe('', () => {
   });
 
   it(
-    'countries/leaderboard',
+    'countries/leaderboard/Europe',
```

Comment:
The capital is unusual in router/url, there are potential problem using this wya.